src/main/java/org/hobbit/core/components/AbstractEvaluationModule.java

```diff
@@ -115,7 +117,7 @@ protected void collectResponses() throws Exception {
             // request next response pair
             props = new BasicProperties.Builder().deliveryMode(2).replyTo(evalStore2EvalModuleQueue.name).build();
             evalModule2EvalStoreQueue.channel.basicPublish("", evalModule2EvalStoreQueue.name, props, requestBody);
-            QueueingConsumer.Delivery delivery = consumer.nextDelivery();
+            Delivery delivery = consumer.getDeliveryQueue().poll(300, TimeUnit.MILLISECONDS);
```

Comment:
What happens here if there is no delivery received after 300ms? To me, this would lead to a `NullPointerException` right? That should be avoided.